querybook/webapp/components/QueryEditor/QueryEditor.tsx

```diff
@@ -149,6 +151,7 @@ export class QueryEditor extends React.PureComponent<
     private codeAnalysis: ICodeAnalysis = null;
     private editor: CodeMirror.Editor = null;
     private autocomplete: SqlAutoCompleter = null;
+    private fetchedTables = new Set<string>();
```

Comment:
i don't think this memo is needed, since fetchDataTableByNameIfNeeded would do the same check as well

Reply:
you're right.